index.html

```diff
@@ -167,7 +177,8 @@ <h4 class="is-3 title">2022</h4>
             gain in-demand skills.</p>
           <h4 class="is-2 title">2023</h4>
           <p>Please take a llok at our plans</p>
-          <table>
+          <!-- WARN: Role is useless, added for HW -->
+          <table role="table">
```

Comment:
Yes, you are completely right, it is useless here. It's unnecessary to specify the role unless you're overriding a non-semantic use. So you can freely remove it